app/web_ui/src/routes/(fullscreen)/setup/(setup)/connect_providers/connect_providers.svelte

```diff
@@ -207,6 +207,20 @@
       api_key_fields: ["API Key", "Base URL"],
       optional_fields: ["Base URL"],
     },
+    {
+      name: "SiliconFlow.cn",
+      id: "siliconflow_cn",
+      description:
+        "SiliconFlow.cn is one of the most popular Chinese model providers.",
+      image: "/images/siliconflow.svg",
```

Comment:
Oh, this will conflict with main. I made a change to have a helper for these URLs since we use them in several places.